setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+setup(name='hetmech',
+      description='A search engine for hetnets',
+      long_description='Matrix implementations of path-count-based measures',
+      url='https://github.com/greenelab/hetmech',
+      license='BSD 3-Clause License',
+      packages=['hetmech'],
+      install_requires=[
+          'hetio>=0.2.8',
+          'numpy',
+          'scipy',
+          'xarray']
```

Comment:
It's ideal to add `,` then newline before closing bracket. This makes sure all diffs to change or add a new dependency in the future only touch a single line